melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/pdl/dto/PDLIdent.java

```diff
@@ -14,4 +14,12 @@
     public boolean erFolkeregisterIdent() {
         return "FOLKEREGISTERIDENT".equals(gruppe);
     }
+
+    public boolean erAktørID() {
+        return "AKTORID".equals(gruppe);
```

Comment:
Det ville være en fordel å definere en enum for gruppene .

Reply:
Ja, jeg var bare usikker på om jeg skulle gjøre det for alle enumer, for å ikke måtte legge til et nytt enum for hver nye verdi som kommer. Men er nok ikke her de fleste endringer kommer 😄

Reply:
Skjønner 😄 Jeg synes det er bra når det ikke er regelmessige endringer (antar det samme som du gjør her) eller når man absolutt må være oppmerksom på endringer. Tror det er litt av begge deler her.